### ISSUE TITLE
Activation of "Show dedicated file columns" should add columns of existing files

### ISSUE THREAD

**Issue by koppor** [koppor/jabref#477]
Click:

![grafik](https://user-images.githubusercontent.com/1366654/102130744-8172f580-3e51-11eb-8ed8-3d71944362d3.png)

Should automatically add all columns existing for the files attached. E.g., if a PDF is attached in any entry in the bib database, the column "PDF" should be added. If Word, then Word, ... - Do not add XLSX if no XLSX is attached

Refs https://github.com/JabRef/jabref/pull/7181

**Comment by ThiloteE** [comment]
@koppor Should I remove this from the good first issues projects group, since you removed the label?

**Comment by koppor** [comment]
@ThiloteE Yes, please remove it. You can also delete our two comments then ^^
